plugins/processors/awsapplicationsignals/internal/resolver/podwatcher.go

```diff
@@ -0,0 +1,198 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+package resolver
+
+import (
+	"sync"
+
+	mapset "github.com/deckarep/golang-set/v2"
+	"go.uber.org/zap"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/tools/cache"
+)
```

Comment:
Assume we'll remove this `podWatcher` after fully switched to `endpointSliceWatcher`, right?

Reply:
Yes, we will remove it when endpointSliceWatcher is mature.